core/gui/src/app/common/type/user.ts

```diff
@@ -42,6 +42,13 @@ export interface Workflow
     workflowName: string;
   }> {}
 
+export interface Dataset
```

Comment:
we already defined such interface. See `core/gui/src/app/common/type/dataset.ts`

Reply:
please use the existing one in your implementation